docs/index.md

```diff
@@ -91,6 +91,32 @@ If a user elects to delete the offline map, the app deletes the offline mobile m
 
 > A user can resume work online without deleting the offline map.
 
+### View and navigate to map bookmarks
```

Comment:
Because "navigate" is an existing verb in the ArcGIS ecosystem, can we go with something else? Perhaps "View and set visible area from map bookmarks"? This probably isn't the best either, but perhaps it's on the right path..?